### PR TITLE
next previous link under article page

### PR DESCRIPTION
**Pull request by gido**
Design:

screen shot 2015-07-10 at 18 13 08

@zufrieden @guillaumeberry Is this OK ?

### CODE REVIEW

File: drupal/sites/all/modules/antistatique_article/antistatique_article.info [hunk 1]
```diff
@@ -6,7 +6,7 @@ project = antistatique_article
 dependencies[] = antistatique_basic_page
 dependencies[] = disqus
 dependencies[] = field_group
-dependencies[] = file_entity
```
Comment (gido) [inline drupal/sites/all/modules/antistatique_article/antistatique_article.info]: @tonifisler I've no idea about this module dependency removal. Is it still used/installed ?

File: drupal/sites/all/modules/antistatique_article/antistatique_article.strongarm.inc [hunk 2]
```diff
@@ -159,6 +159,118 @@ function antistatique_article_strongarm() {
...
+  $strongarm->api_version = 1;
+  $strongarm->name = 'flippy_loop_article';
+  $strongarm->value = 0;
+  $export['flippy_loop_article'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'flippy_next_label_article';
+  $strongarm->value = '[node:title] ›';
```
Comment (gido) [inline drupal/sites/all/modules/antistatique_article/antistatique_article.strongarm.inc]: @zufrieden Do you want more information in the link label ?

**Comment by zufrieden** [comment]
@gido You can merge this first visual version

File: drupal/sites/all/modules/antistatique_article/antistatique_article.info (hunk 1 above)
Comment (tonifisler) [inline drupal/sites/all/modules/antistatique_article/antistatique_article.info]: It's still used yes. This is the module that is transforming files into "nodes". --> `/admin/content/file`

Comment (gido) [inline drupal/sites/all/modules/antistatique_article/antistatique_article.info]: OK. I revert this change.
